### ISSUE TITLE
App distribution

### ISSUE THREAD

**Issue by fajarnuha** [rizmaulana/kotlin-mvvm-covid19#32]
its still disappointing our apps is suspended because of Google doesn't want any covid related app in playstore
lets figure out the best way the other can take benefit from this app

@rizmaulana said he has an idea to distribute in Firebase app distribution
lets discuss

**Comment by rizmaulana** [comment]
It is something like this mas @fajarnuha  https://proandroiddev.com/devops-understanding-and-applying-ci-cd-pipeline-for-android-developers-part-2-ec5b063c01bc

I already configured on my local machine and enable fastlane to distribute to Firebase, but I face some issue when want to apply into Github CI/CD :
1. How to store Google API Keys while our repo is public repo
2. How to store signing key of the app
3. How to attach analytic only on release version
It seems need more powerful CI/CD than Github action, like Jenkins maybe 😅 Any Idea?

**Comment by pandubaraja** [comment]
Nice question @rizmaulana 

I think we should have another repository for tackling question number 1 and 2

Actually we can push commit to multiple repositories. Now the question is can we specified rule inside .gitignore specified for each repository? e.g. on public repo ignore api key and jks file and on private repo don't ignore it.

Here's something to read about one project has multiple repositories
Refs: https://stackoverflow.com/questions/11690709/can-a-project-have-multiple-origins/11690868

**Comment by fajarnuha** [comment]
usually CI/CD have some kind of mechanism to store secret value. doesnt fastlane/github action have it?

having 2 repos will complicate things for this kind of project i think...

for the sake of simplicity lets ignore analytics hit even on debug, we can filter by version number if we want to see release version only

**Comment by pandubaraja** [comment]
> usually CI/CD have some kind of mechanism to store secret value. doesnt fastlane/github action have it?
> 
> having 2 repos will complicate things for this kind of project i think...
> 
> for the sake of simplicity lets ignore analytics hit even on debug, we can filter by version number if we want to see release version only

@fajarnuha you're right about having 2 repos will complicate things cannot find how to do it in the right way too, we can ignore that idea.

This article below can be a good read to start implementing secrets with Github Action
https://help.github.com/en/actions/configuring-and-managing-workflows/using-variables-and-secrets-in-a-workflow

**Comment by fajarnuha** [comment]
just come into my mind, can we use Github release for the start? what do you guys think?
![image](https://user-images.githubusercontent.com/15099703/77074537-91c18580-6a23-11ea-96b2-a0503fc5a390.png)

**Comment by pandubaraja** [comment]
Good idea! Let use release and tag, with this we can list all features shipped on each version as documentation.

**Comment by rizmaulana** [comment]
Good ideas mas @pandubaraja @fajarnuha  we can start for next release to implement this APi : https://indonesia-covid-19-api.now.sh/api

**Comment by fajarnuha** [comment]
nice finding @rizmaulana lets do it!! ill create the issue and lets discuss there

**Comment by fajarnuha** [comment]
> nice finding @rizmaulana lets do it!! ill create the issue and lets discuss there

see #38
